### ISSUE TITLE
Propagate Orthanc logs when system test fails

### ISSUE THREAD

**Issue by milanmlft** [UCLH-Foundry/PIXL#275]
### Definition of Done / Acceptance Criteria

The (relevant parts of the) Orthanc logs are displayed in the CI output when one of the checks fails. 

### Details and Comments

At the moment, some of the system test checks might fail because of something going wrong during the Orthanc processing, but requires re-running locally and manually inspecting the logs to find out what.
Would be nice if we could see this immediately in the CI output.

**Comment by stefpiatek** [comment]
yeah could have a function which runs `docker compose logs` on the production components if there's a non zero exit and then propogates the exit code would be good.

**Comment by milanmlft** [comment]
Not sure how to make Orthanc return an non-zero exit code from the plugin though, I've tried adding `raise SystemExit` in `try`/`except` blocks, e.g. when it fails to connect to the FTP server, but that doesn't make the Orthanc service fail, it just shows up in the logs.

**Comment by milanmlft** [comment]
Done in #301
